### PR TITLE
Mbowerman add sizing

### PR DESCRIPTION
**Pull request by m-d-bowerman**
Adds page about experiment sizing in Mozanalysis to the Getting Started > Data Scientists list.

### CODE REVIEW

File: docs/data-scientists/experiment-sizing.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Experiment sizing in Mozanalysis retrieves historical data from BigQuery in a way that mimicks how experiment enrollment and analysis is conducted by [Jetstream]: an enrollment period is defined, during which clients that satisfy certain target conditions are recorded; following enrollment, metrics are calculated for those clients over a defined analysis period. Following collection of metrics, sample size calculation is performed that corresponds to the statistical tests to be used in analyzing the eventual experiment. 
+
+<img src="/img/mozanalysis/mozanalysis-sizing.png" alt="Mozanalysis sizing workflow" className="img-lg"/>
+
+### Enrollment and analysis
+
+Similar to how experiments use an enrollment period to select clients for a study before collecting metrics for a period of time, Mozanalysis experiment sizing records all clients that satisfy the target conditions during an enrollment period, set by the user based on the analysis start date and the number of dates to enroll. Metrics are then recorded for a user-defined number of days for the analysis.
+
+<img src="/img/mozanalysis/enrollment-period.png" alt="Analysis with an enrollment period" className="img-lg"/>
```
Comment (danielkberry) [inline docs/data-scientists/experiment-sizing.md:19]: I don't think this visual corresponds to what your code is actually doing. For a given client, observation begins immediately after enrollment, as the first analysis windows (for the daily, weekly, monthly levels) will begin at zero days after enrollment. 

 Check out [this image](https://experimenter.info/configuring#visual-explanation) as an example. Basically, the yellow arrows should be shifted back to start at the blue enrollment rectangles.

File: docs/data-scientists/experiment-sizing.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+An alternative to using an enrollment period to select clients is to use continuous enrollment, where any client that satisfies the targeting criteria during the entire analysis period is enrolled; metrics are collected from each client's enrollment date to the end of the analysis period.
+
+<img src="/img/mozanalysis/continuous-enrollment.png" alt="Analysis with continuous enrollment" className="img-lg"/>
```
Comment (danielkberry) [inline docs/data-scientists/experiment-sizing.md:23]: What's the purpose of an enrollment window under a continuously enrolling experiment? Why do we "end enrollment at start date + analysis length" but also enrollment isn't ended?

File: docs/data-scientists/experiment-sizing.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+### Time series of metrics
+
+In some cases, experiments will require client-level time series data. Mozanalysis can split the analysis period into time series periods (either daily, weekly, or monthly).
+
+<img src="/img/mozanalysis/time-series.png" alt="Time series metrics" className="img-lg"/>
```
Comment (danielkberry) [inline docs/data-scientists/experiment-sizing.md:29]: Similar issue as the first diagram, arrows should be shifted

File: docs/data-scientists/experiment-sizing.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Targets and metrics
+
+Selecting clients for analysis is accomplished by using the [`Segment`](https://github.com/mozilla/mozanalysis/tree/main/src/mozanalysis/segments) objects in Mozanalysis; users can either reuse segments that currently exists in Mozanalysis or by defining one at runtime. Segments consist of a data source, either a table or view in BigQuery, and a SELECT expression to filter that data source; this SELECT expression must include a SQL aggregate function. 
+
+Similarly, Mozanalysis experiment sizing reuses `Metric` objects from [Jetstream](jetstream/metrics.md), and users can reuse metrics that are currently implemented in [Mozanalysis](https://github.com/mozilla/mozanalysis/tree/main/src/mozanalysis/metrics) or [jetstream-config], or users can define their own at runtime.
+
+Metrics and target segments are passed to Mozanalysis experiment sizing as lists of `Segment` or `Metric` objects; users may include multiple of each in the analysis.
```
Comment (danielkberry) [inline docs/data-scientists/experiment-sizing.md:43]: Maybe mention here that multiple `Segment`s are ANDed together and if you want a complex targeting expression using ORs, you should write that as a single segment

**Review by danielkberry — CHANGES_REQUESTED** [review]
I noted a few things, mostly about fixing the visuals. The one part that I'm not sure about is how continuously enrolling experiments work here. Maybe you can expand that visual to communicate how users that enroll early will have more analysis windows (for a given analysis level).

File: docs/data-scientists/experiment-sizing.md (hunk 2 above)
Comment (m-d-bowerman) [reply docs/data-scientists/experiment-sizing.md:23]: Basically to set an end date for the analysis, though I see now that this isn't fulfilling the purpose I had in my head: what it's doing now is just using an enrollment period equal to the collection period from the non-continuous enrollment. I'm going to leave it as-is in the documentation, but I'll add to my list making dates for a continuous enrollment analysis perform how we'd expect.

Comment (danielkberry) [reply docs/data-scientists/experiment-sizing.md:23]: Maybe I'm not understanding. How is the experiment end date related to enrollment end date here? Aren't those just separate things?

File: docs/data-scientists/experiment-sizing.md (hunk 1 above)
Comment (m-d-bowerman) [reply docs/data-scientists/experiment-sizing.md:19]: That's my bad, these images were created when sizing _did_ work this way, and I failed to update the images (or proofread)

File: docs/data-scientists/experiment-sizing.md (hunk 2 above)
Comment (m-d-bowerman) [reply docs/data-scientists/experiment-sizing.md:23]: In typical experiments, yeah, they're separate (experiment end is enrollment length + analysis length). In the case of continuous enrollment, experiment end is (first enrollment date + analysis length) + (last enrollment date + analysis length). So the code should be refactored to create a separate study duration and analysis length

**Review by danielkberry — APPROVED** [review]

**Pull request merged**